lib/ansible/module_utils/cloudgenix_util.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2018 CloudGenix Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import os
+
+from ansible.module_utils.six import string_types, binary_type, text_type
+
+try:
+    import cloudgenix
+
+    HAS_CLOUDGENIX = True
+except ImportError:
+    cloudgenix = object()
+    HAS_CLOUDGENIX = False
+
+try:
+    # Although this is to allow Python 3 the ability to use the custom comparison as a key, Python 2.7 also
+    # uses this (and it works as expected). Python 2.6 will trigger the ImportError.
+    # taken from /lib/ansible/module_utils/ec2.py
+    from functools import cmp_to_key
+
+    PY3_COMPARISON = True
+except ImportError:
+    PY3_COMPARISON = False
+
+# try:
+#     from __main__ import display
+# except ImportError:
+#     from ansible.utils.display import Display
+#     display = Display()
+
+
+def cloudgenix_common_arguments():
+    return dict(
+        auth_token=dict(aliases=['x_auth_token'], no_log=True),
+        tenant_id=dict(type='str', required=False),
+        controller=dict(type='str', required=False),
+        ssl_verify=dict(type='bool', required=False, default=True),
```

Comment:
Could you please rename this parameter to `validate_certs` in order to match every other modules in Ansible ?